# HG changeset patch
# User Neil <nyamatongwe@gmail.com>
# Date 1489633522 -39600
# Node ID c3bba4a9193ea55175c309109e5bab8c87f2a0bd
# Parent  2b25287311d015ecc9971c101f0f40ae00c8ca4e
For single phase drawing ensure remainder of line filled by always filling
in tag fold drawing.

diff -r 2b25287311d0 -r c3bba4a9193e src/EditView.cxx
--- a/src/EditView.cxx	Thu Mar 16 14:04:19 2017 +1100
+++ b/src/EditView.cxx	Thu Mar 16 14:05:22 2017 +1100
@@ -1106,7 +1106,7 @@
 		}
 	}
 
-	if ((phasesDraw != phasesOne) && (phase & drawBack)) {
+	if (phase & drawBack) {
 		surface->FillRectangle(rcSegment, textBack);
 
 		// Fill Remainder of the line
